prettier.config.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  // eslint-disable-next-line global-require
+  plugins: [require('prettier-plugin-tailwindcss')],
+};
```

Comment:
Make sure to first merge [this branch](https://github.com/CodersCrew/coderscrew-website/pull/38) if you want to use this lib here

Reply:
Waiting for approval.